src/balatro_gym/game/scoring.py

```diff
@@ -98,13 +98,23 @@ def _get_flush(hand: Sequence[PlayingCard]) -> Sequence[PlayingCard]:
         return hand
     return []
 
+def is_consecutive(ordered_ranks: Sequence[int]) -> bool:
+    prev_rank = ordered_ranks[0]
+    for rank in ordered_ranks[1:]:
+        if not (rank == prev_rank + 1):
+            return False
+        prev_rank = rank
+    return True
 
 def _get_straight(hand: Sequence[PlayingCard]) -> Sequence[PlayingCard]:
+    sorted_ranks = sorted([card.rank.value.order for card in hand])
+    if is_consecutive(sorted_ranks) or _is_royal(hand):
+        return hand
     return []
 
 
 def _is_royal(hand: Sequence[PlayingCard]) -> bool:
-    return all([card.is_face_card() for card in hand])
+    return set([card.rank.value.order for card in hand]) == {0, 9, 10, 11, 12}
```

Comment:
There may be instances where the hand has repeat ranks and still be considered royal. Ex: `hand = [Rank.King]*5`. I think asserting equality on the given set would fail in this case.
```suggestion
    face_cards = set([card.rank.value.order for card in hand])
    return face_cards.issubset({0, 9, 10, 11, 12}) and len(face_cards) ==5
```

Reply:
Also, does `0` exist as an order? Looking at other additions, the lowest order is the `Ace` which is `1`. Is that right?

Reply:
> There may be instances where the hand has repeat ranks and still be considered royal

I don't think there are other instances of royal hands, apart from the royal flush (https://balatrogame.fandom.com/wiki/Poker_Hands). I don't think there's a royal 5 of kind hand.

> Also, does 0 exist as an order? Looking at other additions, the lowest order is the Ace which is 1. Is that right?

Good catch! I ordered started from 0 at first and then changed and forgot to update. This pointed out an issue with `PokerHandType` (since the hand test should not have passed for the royal flush), when doing an equality it looks at the chip and mult to verify the assertion so if we have a hand with the same `PokerScale` then it shows as equal. I updated the code to match all the original scale values which are all different, so I don't think it should be a problem anymore.

Reply:
> I don't think there are other instances of royal hands, apart from the royal flush 

TIL(again) that a royal flush is also a straight. Oops